apps/astarte_housekeeping/priv/migrations/realm/0009_create_ownership_vouchers_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE :keyspace.ownership_vouchers (
+  id uuid,
+  pkey blob,
+  voucher blob,
+  exp_timestamp timestamp,
```

Comment:
can we use whole and explicit names?


```suggestion
  public_key blob,
  voucher_data blob,
  expiry timestamp,
```